package.json

```
@@ -93,6 +95,12 @@
     "yamljs": "^0.3.0"
   },
   "devDependencies": {
+    "jest": "^29.7.0",
+    "eslint": "^8.56.0",
+    "eslint-plugin-jest": "^27.6.0",
+    "eslint-plugin-node": "^11.1.0",
+    "fs-extra": "^11.2.0",
```

Biome (1.9.4)

[error] The key fs-extra was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)